Python/crossfire/crossfire/occurrences.py

```diff
@@ -0,0 +1,52 @@
+from queue import Empty, Queue
+from urllib.parse import urlencode
+
+
+class Occurrences:
+    def __init__(self, client, id_state, id_cities, format):
+        self.client = client
+        self.buffer = Queue()
+        self.id_state = id_state
+        self.id_cities = id_cities
+        self.next_page = 1
+        self.format = format
```

Comment:
Essa lista de parâmetros começa a ficar longa. Podemos pelo menos organizá-la?

Uma forma quase intuitiva é colocar primeiro os argumentos que são basicamente “copiar e colar”, e depois colocar o resto:

```suggestion
        self.client = client
        self.id_state = id_state
        self.id_cities = id_cities
        self.format = format
        
        self.buffer = Queue()
        self.next_page = 1
```

Outra forma seria usar o [`@dataclass` com um `__post_init__`](https://cuducos.me/2018/02/26/um-rapido-tour-sobre-as-classes-de-dados-do-python-3.7.html), ambos nativos do Python.